style.css

```
@@ -1,4 +1,5 @@
-/* Modular CSS entrypoint */
+/* Modular CSS entrypoint — theme.css must load first (tokens before components) */
+@import url("css/theme.css");
```

Stylelint (17.14.0)

[error] Expected "url("css/theme.css")" to be ""css/theme.css"" (import-notation)

(import-notation)